PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
+
+What does this implement/fix? Explain your changes.
+---------------------------------------------------
+…
+
+Does this close any currently open issues?
+------------------------------------------
+…
+
+
+Any relevant logs, error output, etc?
+-------------------------------------
+(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)
+
+Any other comments?
+-------------------
+…
+
+Where has this been tested?
+---------------------------
+**Operating System:** …
+
+**Platform:** …
+
+**Target Platform:** …
+
+**Toolchain Version:** …
+
+**SDK Version:** …
```

Comment:
We don't need a toolchain version or a sdk version neither target plataform.
We should probably just keep the operating system to make things simple.

Reply:
okay, done :)

Reply:
Awesome thanks!